common/src/main/java/com/datastax/oss/common/sink/config/CassandraSinkConfig.java

```diff
@@ -347,6 +352,32 @@ private void deprecatedSecureBundle(Map<String, String> connectorSettings) {
         Function.identity());
   }
 
+  static void decodeBase64EncodedSecureBundle(Map<String, String> javaDriverSettings) {
+    // it is very awkward on k8s to pass ZIP files
```

Comment:
perhaps this comment line could be omitted?

Reply:
removed